<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>910</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Tienda Do√±a Pelos</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="boton1Crear">
    <property name="geometry">
     <rect>
      <x>253</x>
      <y>270</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Crear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="boton2Listar">
    <property name="geometry">
     <rect>
      <x>494</x>
      <y>270</y>
      <width>141</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Listar Productos</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="salida">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>310</y>
      <width>851</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>20</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Codigo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>70</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Nombre</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>120</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Numero de unidades</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>170</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Costo unitario</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>220</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Tipo de producto</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="objCodigo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>20</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objNombre">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>70</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objUnidades">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>120</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="objValorU">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>170</y>
      <width>191</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="objCombo">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>220</y>
      <width>191</width>
      <height>26</height>
     </rect>
    </property>
    <property name="currentText">
     <string>Basico</string>
    </property>
    <item>
     <property name="text">
      <string>Basico</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Aseo</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>General</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>910</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
